File: misc/arithmetics/c/mult_no_mult/mult_no_mult.c
/**
 *
 * Multiply without multiplying algorithm.
 * Notice that there is no control of zero or negative numbers.
 *
 */

#include <stdio.h>

int	main(void)
{
  int a = 3;
  int b = 5;
  int baux = b;
  int result = 0;

  while (baux)
  {
    result += a;
    baux--;
  }

  printf("Result of %d * %d = %d\n", a, b, result);
  return (0);
}

File: misc/arithmetics/javascript/mult_no_mult/mult_no_mult.js
/**
 *
 * Multiply without multiplying algorithm.
 * Notice that there is no control of zero or negative numbers.
 *
 */

const mult_no_mult = (a , b) => {

  let aux = 0;

  for(let i = b; i > 0; i--)
    aux += a;

  console.log(`Result of ${a} * ${b} = ${aux}`);
}

mult_no_mult(3, 6);
